docs/advanced/fields.rst

```diff
@@ -467,18 +474,18 @@ If you're an experienced Python developer, you might wonder why is
         def heavy_method(self):
             # ...
 
-Don't do it! There are two issues with ``lru_cache``, which make it unsuitable
-here:
+Don't do it! There are two issues with :func:`functools.lru_cache`, which make
```

Comment:
Note: You can use InterSphinx but keep the short name if you wish:

```
:func:`~functools.lru_cache`
```

or the more generic (for cases where ~ does not work):

```
:func:`lru_cache <functools.lru_cache>`
```